'

```diff
@@ -0,0 +1,31 @@
+r cd9f003 Drop nullref type
```

Comment:
Stray file added in?

Reply:
Oh, bizarre. I've no idea how my rebase commands got committed. Nice catch!